lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -2633,12 +2634,20 @@ static lldb::ModuleSP GetUnitTestModule(lldb_private::ModuleList &modules) {
   return ModuleSP();
 }
 
-lldb::TypeSystemSP SwiftASTContext::CreateInstance(
-    const SymbolContext &sc,
-    TypeSystemSwiftTypeRefForExpressions &typeref_typesystem,
-    const char *extra_options) {
+lldb::TypeSystemSP
+SwiftASTContext::CreateInstance(const SymbolContext &sc,
```

Comment:
Reading the code, this function is huge and does a lot of different things. We should probably split it up at some point, but not necessarily in this patch.

Reply:
Yes, functions like the triple computation could get factored out for readability.